src/gui/editors/AutomationEditor.cpp

```diff
@@ -1261,6 +1266,48 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 			p.drawLine( x, grid_bottom, x, x_line_end );
 		}
 
+		// draw ghost notes
+		// const NoteVector* currentMidiNotes = &(getGUI()->pianoRoll()->currentMidiClip()->notes());
+		if (m_ghostNotes != nullptr)
+		{
+			const NoteVector& notes = m_ghostNotes->notes();
+			int minKey = 128;
+			int maxKey = 0;
+
+			for (const Note* note : notes)
+			{
+				int noteKey = note->key();
+
+				maxKey = std::max(maxKey, noteKey);
+				minKey = std::min(minKey, noteKey);
+			}
+
+			for (const Note* note : notes)
+			{
+				int len_ticks = note->length();
+
+				if (len_ticks == 0) { continue; }
+				else if (len_ticks < 0) { len_ticks = 4; }
```

Comment:
Might want to add a comment that this line will have to be changed/deleted after #5902 gets merged, but honestly, I think this will get merged first anyways

Reply:
Sure, I wasn't aware of that PR, but I will update this code accordingly once it merges.